index.html

```diff
@@ -309,15 +309,15 @@ <h4>Why make an all-women and non-binary hackathon?</h4>
                     <div class="faq-toggle-icon"></div>
                 </div>
                 <div class="panel">
-                    <p>Women and non-binary people are underrepresented in the tech world, and it’s up to everyone to change that! Technica’s goal is to create a welcoming environment where people from underrepresented genders can build cool projects and share their stories!</p>
+                    <p><b>Women, non-binary, and gender non-conforming people</b> are underrepresented in the tech world, and it’s up to everyone to change that! Technica’s goal is to create a welcoming environment where people from underrepresented genders can build cool projects and share their stories!</p>
                 </div>
                 <div class="accordion">
                     <h4>Do I need to know how to code?</h4>
                     <div class="faq-toggle-icon"></div>
                 </div>
                 <div class="panel">
                     <p>
-                      You don’t need ANY coding experience to participate! If you do, that’s great! If you don’t, that’s also great!
+                      <b>No</b>, you don’t need ANY coding experience to participate! If you do, that’s great! If you don’t, that’s also great! 
```

Comment:
do you think this should appear more bold? i think i'll fix up the fonts to allow for bolder noto sans